### ISSUE TITLE
模型训练报错

### ISSUE THREAD

**Issue by Angus07** [PaddlePaddle/Paddle#21249]
错误信息如下：
![image](https://user-images.githubusercontent.com/4252318/69205952-65395200-0b86-11ea-80da-c8c6c848a4cc.png)
我通过print打印了log，在
                scheduled_lr, loss_scaling = optimization(
                    loss=graph_vars["loss"],
                    warmup_steps=warmup_steps,
                    num_train_steps=max_train_steps,
                    learning_rate=args.learning_rate,
                    train_program=train_program,
                    startup_prog=startup_prog,
                    weight_decay=args.weight_decay,
                    scheduler=args.lr_scheduler,
                    use_fp16=args.use_fp16,
                    use_dynamic_loss_scaling=args.use_dynamic_loss_scaling,
                    init_loss_scaling=args.init_loss_scaling,
                    incr_every_n_steps=args.incr_every_n_steps,
                    decr_every_n_nan_or_inf=args.decr_every_n_nan_or_inf,
                    incr_ratio=args.incr_ratio,
                    decr_ratio=args.decr_ratio)
这一句断了，但是输入的graph_vars["loss"]形状我打印出来和能跑通的形状是一样的，都是
![image](https://user-images.githubusercontent.com/4252318/69206167-e7c21180-0b86-11ea-9e48-f4417324b259.png)
请问这个错误是什么意思呢？我如何定位到错误的原因？

**Comment by MRXLT** [comment]
你好，可以贴一下所使用的paddle版本和相关代码吗，有没有其他的错误信息呢
